changelogs/fragments/470-interfaces-speed-bugfix.yaml

```
@@ -0,0 +1,3 @@
+---
+bugfixes:
+  - sonic_interfaces - Remove conditional error message for speed configuration (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/470).
```

Please describe the specific restriction being removed:

```suggestion
  - sonic_interfaces - Remove the restriction preventing configuration of interface speed for port channel member interfaces (https://github.com/ansible-collections/dellemc.enterprise_sonic/pull/470).
```